static/src/javascripts/projects/commercial/modules/ad-verification/prepare-ad-verification.ts

```diff
@@ -41,35 +43,26 @@ const maybeRefreshBlockedSlotOnce: ConfiantCallback = (
 		},
 	);
 
-	// check if ad is blocked and haven't refreshed the slot yet.
-	if (
-		isBlocked &&
-		!!blockedSlotPath &&
-		!confiantRefreshedSlots.includes(blockedSlotPath)
-	) {
-		setForceSendMetrics(true);
-		captureCommercialMetrics();
-
-		const slots = window.googletag?.pubads().getSlots() ?? [];
-		slots.forEach((currentSlot) => {
-			if (blockedSlotPath === currentSlot.getSlotElementId()) {
-				// refresh the blocked slot to get new ad
-				const advert = getAdvertById(blockedSlotPath);
-				if (!advert) return;
-
-				advert.slot.setTargeting('confiant', String(blockingType));
-
-				setForceSendMetrics(true);
-				captureCommercialMetrics();
-
-				if (shouldRefresh()) {
-					refreshAdvert(advert);
-
-					// mark it as refreshed so it won't refresh multiple time
-					confiantRefreshedSlots.push(blockedSlotPath);
-				}
-			}
-		});
+	// don’t run the logic if the ad is only screened
+	if (!isBlocked || !blockedSlotPath) return;
+
+	const advert = getAdvertById(blockedSlotPath);
+	if (!advert) throw new Error(`No slot found for ${blockedSlotPath}`);
+
+	const eventTimer = new EventTimer();
```

Comment:
Oh no, this is not the correct way to call this. It should have been `EventTimer.get()`.